CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Alertmanager: Updated to latest main. It deprecates Alertmanager APIv1, and replaces all APIv1 endpoints with a deprecation notice. You should replace all use of the /v1/alerts, /v1/silences, /v1/receivers and /v1/status endpoints with their APIv2 equivalent.
```

Comment:
Ditto from marco's suggestion.

- We don't use wording such as "Updated to latest main"; we simply list what the changes do for the end user.
- I'd recommend a list the endpoints directly.

```suggestion
* [CHANGE] Alertmanager: Deprecates the Alertmanager `v1` API. All `v1` API endpoints now respond with a JSON deprecation notice and a status code of `410`. All endpoints have a `v2` equivalent. The list of endpoints is:
  * `<alertmanager-web.external-url>/api/v1/alerts`
  * `<alertmanager-web.external-url>/api/v1/silence`
  * `<alertmanager-web.external-url>/api/v1/silences`
  * `<alertmanager-web.external-url>/api/v1/status`
  * `<alertmanager-web.external-url>/api/v1/receivers`
  * `<alertmanager-web.external-url>/api/v1/alerts`
```